src/App.js

```diff
@@ -11,14 +11,16 @@ import HomeScreen from './components/HomeScreen/HomeScreen';
 import Call from './components/Call/Call';
 import Header from './components/Header/Header';
 import Tray from './components/Tray/Tray';
+import HairCheck from './components/HairCheck/HairCheck';
```

Comment:
I don't think it's added in this PR but I noticed in the error message, we're using backticks but should probably highlight it somehow with CSS instead. Or link to the file in Github or something. Whatever you think. 👍 
![CleanShot 2022-05-13 at 14 17 57@2x](https://user-images.githubusercontent.com/18265845/168343747-caa75c83-9953-4ff5-9283-79d4f1777238.png)

Reply:
TBH the whole error handling could use some extra love -- if you try to join a room with an invalid URL now it happily goes to the haircheck, since it's technically not an API error 😅  I think I'll combine this feedback with Liza's earlier comment [here](https://github.com/daily-demos/custom-video-daily-react-hooks/pull/4#issuecomment-1126027319) and create a new PR for this.